lib/dependabot/file_parsers/elm/elm_package.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require "dependabot/dependency"
+require "dependabot/file_parsers/base"
+require "dependabot/file_fetchers/elm/elm_package"
+require "dependabot/shared_helpers"
+require "dependabot/utils/elm/version"
+
+module Dependabot
+  module FileParsers
+    module Elm
+      class ElmPackage < Dependabot::FileParsers::Base
+        require "dependabot/file_parsers/base/dependency_set"
+
+        MAX_VERSION = 9999
+        REQUIREMENT_REGEX =
+          /(?<operator><=?)\s*(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)$/
+
+        def parse
+          dependency_set = DependencySet.new
+
+          parsed_package_file.fetch("dependencies").each do |name, req|
+            dependency_set <<
+              Dependency.new(
+                name: name,
+                version: max_version_for(req),
+                requirements: [{
+                  requirement: req, # 4.0 <= v <= 4.0
+                  groups: nil, # we don't have this (its dev vs non-dev)
+                  source: nil, # elm-package only has elm-package sources
+                  file: "elm-package.json"
+                }],
+                package_manager: "elm-package"
+              )
+          end
+
+          dependency_set.dependencies.sort_by(&:name)
+        end
+
+        private
+
+        def check_required_files
+          raise "No elm-package.json!" unless elm_package_file
+        end
+
+        def max_version_for(version_requirement)
```

Comment:
Note to self: we might be able to do away with this - Dependabot can handle updating libraries in other languages, when the parser returns `nil` for the version (because a range is specified).